livable-uml/src/pages/editor/toolbar/AddIconAction.ts

```diff
@@ -0,0 +1,4 @@
+
+export default function addIconAction() {
```

Comment:
I think the general naming convention when creating a functional component is capitalizing the beginning like **AddIconAction**

If this were just a function name within another component, then this naming convention is right.